modules/media_entity/instagram.md

```diff
@@ -3,3 +3,70 @@
 Implements media entity resource provider for Instagram photos and videos.
 
 Project page: (https://drupal.org/project/media_entity_instagram)
+
+## Requirements
+
+1. [Media Entity](https://www.drupal.org/project/media_entity)
+
+## Installation
+
+1. Download [Media Entity](https://www.drupal.org/project/media_entity) from Drupal.org.
+2. Download [Media Entity Instagram](https://drupal.org/project/media_entity_instagram) from Drupal.org.
+3. Install both Media Entity Instagram and Media Entity in the [usual way](https://www.drupal.org/documentation/install/modules-themes/modules-8).
+
+## Overview
+
+Instagram field formatter can be used with three different types of fields:
+* Text plain fields `string_long`, `string_short`
+* Link fields
+* Entity reference that references appropriate media bundles
+
+In the **Usage** section configuration of media bundles and supported field types will be described in detail.
+
+## Usage
+
+1. Create a Media bundle
+  * On `admin/structure` choose **Media bundles**
+    ![Step 1](images/instagram/step_1.png)
+  * Click on **+ Add media bundle**
+    ![Step 2](images/instagram/step_2.png)
+  * Fill *Label*, *Description* and *Type provider fields* for your media bundle and click **Save media bundle**
+    ![Step 3](images/instagram/step_3.png)
+2. Create an Instagram link field on a Media bundle
+ * On media bundles overview page choose **Manage fields** on created **Instagram** bundle.
+   ![Step 4](images/instagram/step_4.png)
+ * Click on **+ Add field**. For a storage type choose **Link**, fill a *Label* field and click **Save and continue**.
+   ![Step 5](images/instagram/step_5.png)
```

Comment:
One step is missing here. We need to go back to bundle edit page and select this field in the "Field with source information" dropdown.